internal/controller/config/config.go

```diff
@@ -39,16 +39,16 @@ func JobName(uuid string) string {
 
 type Config struct {
 	AgentTokenSecret string `mapstructure:"agent-token-secret" validate:"required"`
-	BuildkiteToken   string `mapstructure:"buildkite-token" validate:"required"`
+	BuildkiteToken   string `mapstructure:"buildkite-token"    validate:"required"`
 	Debug            bool
-	Image            string        `validate:"required"`
+	Image            string        `                                  validate:"required"`
```

Comment:
Is `mapstructure` needed before `validate`? Perhaps they could be swapped, to avoid the huge blanks.

Reply:
There's a case where there's no `validate` tag, but there is a `mapstructure` tag. Rather bizarrely, mapstructure is case-insensitive: https://pkg.go.dev/github.com/mitchellh/mapstructure?utm_source=godoc#hdr-Field_Tags.

So I think it's best to always explicitly specify lowercase kebab case field names. While this could break something, we only use these for integration with viper and cobra in cmd/controller/controller.go, and there, all the field names are lower kebab case.